config/sidebar.yml

```diff
@@ -641,8 +641,22 @@ articles:
         url: /custom-domains
         children:
 
-          - title: Custom Domains with Self-Managed Certificates
+          - title: Auth0-Managed Certificates
+            url: /custom-domains/auth0-managed-certificates
+
+          - title: Self-Managed Certificates
             url: /custom-domains/self-managed-certificates
+            children:
+
+              - title: Reverse Proxy Example
```

Comment:
Should probably link to custom-domains/set-up-azure-cdn + custom-domains/set-up-cloudfront

Reply:
Got it